docs/contributing/translator_onboarding.md

```diff
@@ -0,0 +1,35 @@
+# ONBOADING GUIDE FOR TRANSLATORS
+
+   Wagtail uses transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
+## Translation workflow overview
+   Each new release has about two weeks release candidate period. At the start of that period the translations are put into transifex, we notify the channel in slack that new translations are needed. Those are then updated when there is a release candidate until the final release. If translations reach a threshold of X% ( this is about 90) we communicate that this is an officially supported translation in the docs.
+
+
+## What is transifex
+
+ Wagtail is localized (translated) using [transifex](https://www.transifex.com/). A software and web localization platform that allows organisations to coordinate translation projects and collaborate with communities to translate materials. 
+
+
+## To help out in translating wagtail here's what you will do
+
+- Introduce yourself on [slack]( https://wagtailcms.slack.com/archives/CTKN6UXKN) to join  fellow translators on slack.
+- Request access to become a member of the language team you want to work with on slack. Mention your transifex username when you make this request.
+-	Go to [transifex](https://www.transifex.com/).

```

Comment:
put all these as a flat list with the same indentation.

Also, no need for a period (full stop) at the end of each list item.

Reply:
@lb- I am not familiar with what a flat list is can you please explain?